core/src/main/resources/org/apache/spark/ui/static/webui.js

```diff
@@ -80,4 +80,6 @@ $(function() {
   collapseTablePageLoad('collapse-aggregated-poolActiveStages','aggregated-poolActiveStages');
   collapseTablePageLoad('collapse-aggregated-tasks','aggregated-tasks');
   collapseTablePageLoad('collapse-aggregated-rdds','aggregated-rdds');
+  collapseTablePageLoad('collapse-aggregated-activeBatches','aggregated-activeBatches');
```

Comment:
Probably should not collapse the active batches; they are important and won't be numerous.

CC @ajbozarth

Reply:
I would agree there's no reason to, but this simply enables user to if they want. And I would prefer the uniform look and feel of have all tables collapsible like the other pages.

Reply:
Oh I see. This doesn't also collapse by default? I wondered because of what the name "collapseTablePageLoad" seemed to suggest. Sure, the capability is fine.

Reply:
This function just makes sure to persist collapsed tables on page reload